src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_editor.cljs

```diff
@@ -100,6 +100,66 @@
                                               (swap! state assoc :deleting? true)
                                               (react/call :rm-mc this))}])})
 
+(defn- clear-overlay [state refs]
+       (swap! state assoc :publishing? false))
+
+(defn- render-modal-publish [state refs props]
+       (react/create-element
+         [:div {}
+          [:div {:style {:backgroundColor "#fff"
+                         :borderBottom (str "1px solid " (:line-gray style/colors))
+                         :padding "20px 48px 18px"
+                         :fontSize "137%" :fontWeight 400 :lineHeight 1}}
+           "Publish Method Configuration"]
+          [:div {:style {:padding "22px 48px 40px" :backgroundColor (:background-gray style/colors)}}
+           (style/create-form-label "Method Configuration Namespace")
+           (style/create-text-field {:style {:width "100%"} :ref "mcNamespace"
+                                     :defaultValue (get-in (:config props) ["namespace"])})
+           (style/create-form-label "Method Configuration Name")
+           (style/create-text-field {:style {:width "100%"} :ref "mcName"
+                                     :defaultValue (get-in (:config props) ["name"])})
+           [:div {:style {:marginTop 40 :textAlign "center"}}
+            [:a {:style {:marginRight 27 :marginTop 2 :padding "0.5em"
+                         :display "inline-block"
+                         :fontSize "106%" :fontWeight 500 :textDecoration "none"
+                         :color (:button-blue style/colors)}
+                 :href "javascript:;"
+                 :onClick #(clear-overlay state refs)
+                 :onKeyDown (common/create-key-handler [:space :enter] #(clear-overlay state refs))}
+             "Cancel"]
+            (when (:creating-wf @state)
+                  [comps/Blocker {:banner "Publishing Method Configuration..."}])
```

Comment:
This blocker is in a weird spot in the DOM, though it doesn't really matter.  I would just stick it at the top (i.e., on line 109), or at the bottom so it's out of the way.